components/javascript/toast.js

```diff
@@ -0,0 +1,31 @@
+var topToast = document.querySelector("#top-toast-id");
+var topToastContent = document.querySelector('#top-content');
+var topToastClose = document.querySelector('#top-toast-close');
+
+var bottomToast = document.querySelector('#bottom-toast-id');
+var bottomToastContent = document.querySelector('#bottom-content');
+var bottomToastClose = document.querySelector('#bottom-toast-close');
```

Comment:
Use const and let. Try to avoid var.
Switch to ES6+ !

Reply:
Updated code in ES6.